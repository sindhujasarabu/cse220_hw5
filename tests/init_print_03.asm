.data
_id: .word 4194303
_credits: .word 1023
_name: .asciiz "This is supposed to be a long test case"

.include "p12_tester.asm"
