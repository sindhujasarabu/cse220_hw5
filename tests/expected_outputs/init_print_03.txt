4194303 1023 This is supposed to be a long test case